rpc_proxy.go

```diff
@@ -38,31 +37,29 @@ var (
 	// maxMsgRecvSize is the largest message our proxy will receive. We
 	// set this to 200MiB atm.
 	maxMsgRecvSize = grpc.MaxCallRecvMsgSize(1 * 1024 * 1024 * 200)
-	configPath     = filepath.Join(
-		btcutil.AppDataDir("tdex-dashboard", false), "config.json",
+
+	// flag --addr to customize host:port on which the HTTP server should listen.
+	httpServerAddr = flag.String(
+		"addr",
```

Comment:
```suggestion
		"insecure-httplisten",
```

to be more explicit in what this will result